### ISSUE TITLE
[Error] `DialogContent` requires a `DialogTitle` for the component to be accessible for screen reader users.

### ISSUE THREAD

**Issue by ouestlabs** [emilkowalski/vaul#523]
[Error] `DialogContent` requires a `DialogTitle` for the component to be accessible for screen reader users.

If you want to hide the `DialogTitle`, you can wrap it with our VisuallyHidden component.

For more information, see https://radix-ui.com/primitives/docs/components/dialog

[Warning] Warning: Missing `Description` or `aria-describedby={undefined}` for {DialogContent}.

**Comment by emilkowalski** [comment]
That's intended. You need to provide a title for it to be accessible.
